packages/data/src/lookitS3.ts

```diff
@@ -132,16 +146,18 @@ export default class {
       throw Error("No upload id");
     }
```

Comment:
It might be nice to create a custom error.  I have an example in another package.

Reply:
Looks like you fixed this one here 26d37c631460035db744070209d0358cfaaf1f1d 👍  I don't see any other generic errors, so I'll mark this as resolved.